includes/socket/ServerSocket.hpp

```diff
@@ -13,8 +13,6 @@ class ServerSocket : public TCPSocket {
         ServerSocket(smt::shared_ptr<ServerBlock> config);
         ~ServerSocket(void);
 
-    private:
```

Comment:
Should this be public?

Reply:
I made a lot of things public throughout the project with no better reason than it made our life much easier. The code is much more easy to understand as well. Other than that its the fact that I also could not find a better reason not to... But I'm open to suggestions

Reply:
Keeping it private improves encapsulation. Classes that depend on this one should use it's "interface" and not have to worry about implementation logic. Therefor, if we want to change the implementation in the future, dependent classes won't break as easily. Although I do agree that it makes our live easier, just some food for thought 🤓

Reply:
So what do you suggest? Everything private and getters and setters? Cause if so, @J0Santos has homework :nerd_face:

Reply:
Ahah that could be the case, but maybe there's no need... I just think when we find the need to access the method or attribute's directly, and therefor "break" encapsulation, that it is a good sign that we might need to do some refactoring in the future

Reply:
Resolving this conversation as this is easily fixed in the future and is subjective to personal design preferences